Gaming The Syllabus/Assets/Scripts/Combat/CombatUnit.cs

```diff
@@ -20,9 +20,10 @@ public class CombatUnit : IXmlSerializable
 
     public bool IsAlive { get; private set; } = true;
 
-    public void PerformAction(int actionIndex) 
+    public void PerformAction(int actionIndex)
     {
-        if (IsAlive) {
+        if (IsAlive)
+        {
```

Comment:
I like how you like to keep our brackets consistent. Consistency is always good.